releases/katello/4.14/settings

```diff
@@ -0,0 +1,4 @@
+FULLVERSION='4.14.0-rc1'
+FOREMAN_VERSION='3.12'
+OSES="el8 el9"
```

Comment:
I never thought about this before, but this will pretty much always match Foreman so we can drop it from Katello. https://github.com/theforeman/theforeman-rel-eng/pull/396 cleans it up.